namada-public-testnet-15/Jascai.toml

```
@@ -0,0 +1,16 @@
+[validator.Jascai]
+consensus_public_key = "0089c907996d3ab8eeefad48f899a8388af3a4da598ce389d532db456c604dbff0"
+eth_cold_key = "010275123093aaacb2f74b83955a82f5812e2da41fc36773f21c4cc0c5f7a7f64b87"
+eth_hot_key = "0102e089c56f786f6f369e4b69235ff52cc97f9f48d9fb37f1722d872f169c4e28b0"
+account_public_key = "00ec795d4830a5ee73189c159a1b42d9f88cd55386fa1e1bdab6d7ad05fdc69bf4"
+protocol_public_key = "00c485b757bb214fdaa9d02f30fe1671ae0e13db24740d8b7653c40494582743ad"
+dkg_public_key = "60000000b2b5fc4491f6dc3014b736fafc4b3e5fffa585c695240b4ea82f6969f127ee3531cff5c1ae758154be3852cb607ced05f03a20b61661a40af3a71796daed17039b26a3799e84069b44e1cf5792a2ff082a806680e5a2a5ca839661d00da40201"
+commission_rate = "0.05"
+max_commission_rate_change = "0.01"
+net_address = "194.163.139.165:26656"
+tendermint_node_key = "00c5fbe64479f8d0d4b30a50dcabe2ec75c6df16b3adde01508fface627788cb76"
+email = "urtaev1993@gmail.com"
+discord = ".mr_pool"
+elements = ""
+telegram = "https://t.me/mr_jascai"
+twitter = ""
```

```suggestion
twitter = ""

```
Just add a line after the final line in order to be fully correct, then can merge

---

I rechecked the file, it should be fine now.